datafusion/expr/src/test/function_stub.rs

```diff
@@ -252,6 +252,10 @@ impl AggregateUDFImpl for Count {
         Ok(DataType::Int64)
     }
 
+    fn is_nullable(&self) -> bool {
```

Comment:
BTW this makes the count function stub match the actual count function which is also not nullable

This was required to get the existing tests to pass